internal/output/table.go

```diff
@@ -6,8 +6,8 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/google/osv-scanner/internal/osv"
 	"github.com/google/osv-scanner/pkg/models"
+	"github.com/google/osv-scanner/pkg/osv/v1"
```

Comment:
I'm not sure including "v1" here is idiomatic. I believe that's always implied. 

See https://go.dev/doc/modules/release-workflow#breaking, which talks about how to publish breaking changes to APIs.